pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools", "wheel"]
+requires = ["setuptools"]
```

Comment:
Setuptools docs originally said you should include `wheel`, but it now says to remove it and that they made a mistake.